.gitignore

```diff
@@ -1 +1,3 @@
-log
+/gemfiles/.bundle
+/log
+/vendor/bundle
```

Comment:
if you add these to your global gitignore then you never need to add them to projects

Reply:
Good idea (done). Though we can't assume that anyone who checks out the project will have done so (and we don't want them `git add .`ing them by mistake).